packages/pluggableWidgets/accordion-web/src/Accordion.editorConfig.ts

```diff
@@ -105,3 +109,120 @@ export function check(values: AccordionPreviewProps): Problem[] {
 
     return errors;
 }
+
+export function getPreview(values: AccordionPreviewProps): StructurePreviewProps | null {
+    return {
+        type: "Container",
+        borders: true,
+        children: values.groups
+            .map((group, index): StructurePreviewProps[] => {
+                let headerTextFontSize = 0;
+                let headerTextIconPadding = 0;
+
+                if (group.headerRenderMode === "text") {
+                    switch (group.headerHeading) {
+                        case "headingOne":
+                            headerTextFontSize = 13;
+                            headerTextIconPadding = 18;
+                            break;
+                        case "headingTwo":
+                            headerTextFontSize = 12;
+                            headerTextIconPadding = 18;
+                            break;
+                        case "headingThree":
+                            headerTextFontSize = 11;
+                            headerTextIconPadding = 16;
+                            break;
+                        case "headingFour":
+                            headerTextFontSize = 10;
+                            headerTextIconPadding = 15;
+                            break;
+                        case "headingFive":
+                            headerTextFontSize = 9;
+                            headerTextIconPadding = 14;
+                            break;
+                        case "headingSix":
+                            headerTextFontSize = 8;
+                            headerTextIconPadding = 13;
+                            break;
+                    }
+                }
```

Comment:
Few ways to clean this up:

- You could make this an object with the keys of the `headerHeading` enum and make this a lookup.
- Make a new function that accepts the group and returns a pair of `{ headerTextFontSize, headerTextIconPadding}`

Moving this logic out into either the `getPreview` or module scope should help with making this function less long and more readable already.